shared/pkg/utils/trivy/sourcetypes.go

```diff
@@ -36,3 +42,60 @@ func KubeclaritySourceToTrivySource(sourceType utils.SourceType) (artifact.Targe
 	}
 	return artifact.TargetKind("Unknown"), fmt.Errorf("unable to convert source type %v to trivy type", sourceType)
 }
+
+// nolint:cyclop
+func UntarToDirectory(tarPath, destDirectory string) error {
```

Comment:
can't we use https://pkg.go.dev/golang.org/x/build/internal/untar?

Reply:
or https://github.com/anchore/stereoscope/blob/main/pkg/file/tarutil.go#L128?

Reply:
I didn't try but I assumed because that is an "internal" package that it is not importable from outside of golang.org/x/build

Reply:
so we can use https://github.com/anchore/stereoscope/blob/main/pkg/file/tarutil.go#L128?